app/views/layouts/avalon.html.erb

```diff
@@ -53,13 +53,14 @@ Unless required by applicable law or agreed to in writing, software distributed
     <a id="skip-to-content" class="sr-only sr-only-focusable" href="#content">Skip to main content</a>
     <%= render partial: 'shared/ajax_modal' %>
     
-    <% if !current_page?(root_path) || request.query_string.present? %>
+    <% if !current_page?(root_path) %>
         <%= render 'modules/header' %>
         <%= render :partial => 'modules/global_navigation' %>
         <div class="container-fluid secondary">
     <% end %>  
       <%= render 'modules/flash_messages' %>
         <%= yield %>
+        </div>
```

Comment:
Okay, new problem now: if the current page is the root path, this closing tag for the div doesn't have a matching opening tag.